src/consensus/tx_verify.cpp

```diff
@@ -74,7 +74,9 @@ std::pair<int, int64_t> CalculateSequenceLocks(const CTransaction &tx, int flags
         int nCoinHeight = prevHeights[txinIndex];
 
         if (txin.nSequence & CTxIn::SEQUENCE_LOCKTIME_TYPE_FLAG) {
-            int64_t nCoinTime = block.GetAncestor(std::max(nCoinHeight-1, 0))->GetMedianTimePast();
```

Comment:
If you don't use the result, you can just inline `Assert(bla)->`...

Reply:
Thank you, fixed it